dspace/config/spring/api/discovery.xml

```diff
@@ -55,6 +55,7 @@
                 <entry key="workspace" value-ref="workspaceConfiguration" />
                 <entry key="workflow" value-ref="workflowConfiguration" />
                 <entry key="undiscoverable" value-ref="unDiscoverableItems" />
+                <entry key="discoverableAndUndiscoverableItems" value-ref="discoverableAndUndiscoverableItems" />
```

Comment:
I would prefer a shorter name, what about allitems or administrativeitemsview ?

Reply:
I'd prefer not to use allitems to avoid situations similar to ItemService containing findAll and findAllUnfiltered. The administrativeItemsView seems ok, but it's a name that's not really visible to the user (Angular will use it for the search private items UI, but the user won't see the name)

Reply:
@abollini and @benbosman : I'd recommend dropping "Items" from the key here, as no other filter uses that.  I'm fine with either "discoverableAndUndiscoverable" or "administrativeView".  I don't mind this key being longer, as long as it's descriptive.

Reply:
Shall we use administrativeView in that case, seems to match everyone's requirements?

Reply:
@benbosman : "administrativeView" is fine by me.